stacklight_tests/tests/prometheus/test_alerts.py

```diff
@@ -36,6 +36,36 @@ def check_status(is_fired=True):
             check_status()
         check_status(is_fired=False)
 
+    def test_system_mem_alert(self, cluster, prometheus_alerting):
+        """Check that operation system alerts can be fired.
+         Scenario:
+            1. Check that alert is not fired
+            2. Install 'stress' package
+            3. start process which will to load mem
+            4. Wait until and check that alert was fired
+            5. Remove 'stress' package
+            6. Wait until and check that alert was ended
+
+        Duration 15m
+        """
+        cmp = cluster.filter_by_role("compute")[0]
+        criteria = {
+            "name": "AvgMemAvailablePercent",
+            "service": "system",
+            }
+        cmp.os.apt_get_install_package("stress")
+        prometheus_alerting.check_alert_status(
+            criteria, is_fired=False, timeout=10 * 60)
+        command = "stress --vm-bytes $(awk '/MemFree/{printf \"%d\"," \
+                  " $2 * 0.95;}' < /proc/meminfo)k --vm-keep -m 1" \
+                  " --timeout 600"
```

Comment:
Split it to several calls, in first get output of /proc/meminfo, then get free mem via python, not awk.
From my pov better way is to launch stress in a background via `nohup`.